doc/aos_board.cfg

```diff
@@ -0,0 +1,23 @@
+{
+  "resources": [
+    {
+      "name": "vis",
+      "hosts": [
+        {
+          "ip": "10.0.0.7",
+          "hostname": "wwwivi"
+        }
+      ]
+    }
+  ],
+  "components": [
+    {
+      "id": "rootfs"
+    },
+    {
+      "id": "boot"
+    }
+  ],
+  "vendorVersion": "9",
```

Comment:
remove vendorVersion and vendorVersion fields.
Add board configuration file as separate commit

Reply:
done